test/util/factories/ride_factory.dart

```diff
@@ -54,7 +54,7 @@ class RideFactory extends TripFactory<Ride> {
       startTime: startTime ?? DateTime.now(),
       end: end ?? faker.address.city(),
       endPosition: endPosition ?? Position(faker.geo.latitude(), faker.geo.longitude()),
-      endTime: endTime ?? DateTime.now(),
+      endTime: endTime ?? DateTime.now().add(const Duration(hours: 1)),
```

Comment:
this will be replaced once PR 107 is merged. I needed it to not have failing tests all the time